File: app/src/main/java/com/bunbeauty/papakarlo/feature/create_order/model/OrderCreationUI.kt
package com.bunbeauty.papakarlo.feature.create_order.model

data class OrderCreationUI(
    val isDelivery: Boolean,
    val address: String?,
    val comment: String?,
    val deferredTime: String,
    val totalCost: String?,
    val deliveryCost: String?,
    val amountToPay: String?,
    val isLoading: Boolean,
) {
    val switcherPosition = if (isDelivery) {
        0
    } else {
        1
    }
}

File: iosApp/iosApp/feature/create_order/OrderCreationUI.swift
//
//  OrderCreationUI.swift
//  PapaKarloSwift
//
//  Created by Марк Шавловский on 27.03.2022.
//

import Foundation

struct OrderCreationUI {
    let isDelivery:Bool
    let address:String?
    let comment: String?
    let deferredTime: String
    let totalCost: String
    let deliveryCost: String
    let amountToPay: String
    let isLoading: Bool
    
    
    func switchPosition(isDelivery:Bool) -> Int {
        if (isDelivery) {
            return 0
        } else {
            return 1
        }
    }
}
